src/definitions/behaviors/form.js

```diff
@@ -1486,6 +1540,42 @@ $.fn.form.settings = {
     }
   },
 
+  formatter: {
+    date: function(date) {
+      var date = new Date(date);
```

Comment:
Why is this necessary here? It's not used in the other formatters? :thinking:

Reply:
Mmmmh. I defined it during my tests, and did not implement it in other formatters. The `Intl.DateTimeFormat.prototype.format()` seem to need a date as input, but looks like he perfectly handle a date string. So, should I remove `var date = new Date(date);` from this one or implement a Date object in other ?

Reply:
In Line 721 you added 
```javascript
var date = $calendar.calendar('get date');
```
which will already return a valid Date Object (or null/undefined) and is passed to all the formatters, so you can remove the `var date=new Date(date)`

Reply:
As said, you can safely remove the ´var date=new Date(date)` line.
I would love to approve this PR right after that 😉

Reply:
Done.